### DISCUSSION TITLE
Add AKS clusters in different VNETS to single ArgoCD UI

### DISCUSSION THREAD

**Discussion by raviumishra** [argoproj/argo-cd#19312] [Q&A]
We have 2 **private** AKS clusters [dev & prd] in dedicated VNETS. Have installed ArgoCD in DEV cluster. Want to add prod in the same instance.
![image](https://github.com/user-attachments/assets/e26ee000-85d3-4b16-ab49-597be28539fd)

**Comment by sengkocktan** [comment]
you may need peer the vnet and also add the virtual network link private dns in prod

**Reply by sengkocktan** [reply]
happen to view ---> https://www.youtube.com/watch?v=El7zRS5w0dc  the guy did mention status unknown is normal till you have deploy the app then it will change.
